DEVELOPMENT.md

```diff
@@ -46,6 +46,19 @@ To run a specific e2e test locally:
 $ make e2e-local TEST=TestCreateInstallPlanManualApproval
 ```
 
+##### Updating test images
+
+Sometimes you will need to update the index or bundle images used in the unit or e2e tests. To update those images, you can use the following 2 scripts in the ./scripts/ directory:
+
+```sh
+./scripts/build_test_images.sh
+./scripts/build_e2e_test_images.sh
+```
```

Comment:
And if we are documenting these scripts, might as well write a short summary of what each script does.

Reply:
Done